docs/src/main/sphinx/admin/properties-http-client.rst

```diff
@@ -0,0 +1,312 @@
+======================
+HTTP client properties
+======================
+
+``http-client.https.hostname-verification``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Verifies that the server hostname matches the server certificate.
+
+``http-client.http2.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:**
+
+.. No default value?
+
+Enables the HTTP/2 transport.
+
+``http-client.connect-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Minimum value:** ``0ms``
+* **Default value:** ``5s``
+
+``http-client.request-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Minimum value:** ``0ms``
+* **Default value:** ``5m``
+
+``http-client.idle-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Minimum value:** ``0ms``
+* **Default value:** ``1m``
+
+``http-client.max-connections``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Minimum value:** ``1M``
+* **Default value:** ``200``
+
+``http-client.max-connections-per-server``
```

Comment:
This one is pretty self explanatory, but it's got that same caveat that specifying a minimum value might be a little weird. And I'm not sure what 1M does for an integer prop type?

Reply:
Yeah. Including 1M was a mistake. It's not even a value for this property.